src/main/java/no/nav/familie/integrasjoner/modiacontextholder/ModiaContextHolderController.kt

```diff
@@ -0,0 +1,53 @@
+package no.nav.familie.integrasjoner.modiacontextholder
+
+import no.nav.familie.integrasjoner.client.rest.ModiaContextHolderClient
+import no.nav.familie.integrasjoner.modiacontextholder.domene.ModiaContextHolderNyAktivBrukerDto
+import no.nav.familie.integrasjoner.modiacontextholder.domene.ModiaContextHolderResponse
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.familie.kontrakter.felles.Ressurs.Companion.failure
+import no.nav.familie.kontrakter.felles.Ressurs.Companion.success
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@ProtectedWithClaims(issuer = "azuread")
+@RequestMapping("/api/modia-context-holder")
+class ModiaContextHolderController(
+    private val modiaContextHolderClient: ModiaContextHolderClient,
+) {
+    @GetMapping(
+        produces = [MediaType.APPLICATION_JSON_VALUE],
+    )
+    fun hentContext(): ResponseEntity<Ressurs<ModiaContextHolderResponse>> =
+        ResponseEntity.ok(
```

Comment:
Denne vil jo kaste 200 OK samme hva som skjer. Hvorfor? For meg et det totalt ulogisk at en integrasjonstjeneste returnerer 200 OK når det er feil.